.github/workflows/vita-ci.yml

```diff
@@ -0,0 +1,152 @@
+            deps_cmdline: "sudo apt-get update -qq && sudo apt-get install -qq cmake ninja-build cmake ninja-build lftp git-core python zip",
+            generator: "Unix Makefiles",
+            build_type: "MinSizeRel",
+            executable_name: "thextech",
+            assets_url: "https://wohlsoft.ru/TheXTech/_downloads/thextech-adventure-of-demo-assets-full.7z",
+            subdir_name: "thextech-vita",
+            upload_directory: "www/vita/"
+          }
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set output
+      id: vars
+      run: echo ::set-output name=short_ref::${GITHUB_REF#refs/*/}
+
+    - name: Install Dependencies
+      shell: bash
+      run: |
+        if [[ ! -z "${{ matrix.config.deps_cmdline }}" ]]; then
+          eval ${{ matrix.config.deps_cmdline }}
+        fi
+        # Pull submodules
+        git submodule init
+        git submodule update --recursive --remote
+
+    - name: Install Vita SDK
+      run: |
+        cd ..
+        export PATH=$VITASDK/bin:$PATH
+        git clone https://github.com/vitasdk/vdpm
+        cd vdpm
+        ./bootstrap-vitasdk.sh
+        ./install-all.sh
+
+# NOTE: 2/13/2022 Current Vita version does not use cglm or vitaGL renderer. All is handled by SDL2 at the moment.
+#    - name: Install cglm
+#      run: |
+#        cd ..
+#        export PATH=$VITASDK/bin:$PATH
+#        git clone https://github.com/recp/cglm.git
+#        cd cglm
+#        sed -i "s|-Werror||g" CMakeLists.txt
+#        cmake -B build-vita -G Ninja -DCMAKE_INSTALL_PREFIX=$VITASDK/arm-vita-eabi -DCGLM_STATIC=ON -DCGLM_SHARED=OFF -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_TOOLCHAIN_FILE=/usr/local/vitasdk/share/vita.toolchain.cmake .
+#        cmake --build build-vita --target all
+#        cmake --build build-vita --target install
+
+#    - name: Install Latest vitaGL
+#      run: |
+#        cd ..
+#        export PATH=$VITASDK/bin:$PATH
+#        git clone https://github.com/Rinnegatamante/vitaGL.git
+#        cd vitaGL
+#        NO_DEBUG=1 NO_TEX_COMBINER=1 make
+#        make install
+
+#    - name: Download assets
+#      uses: carlosperate/download-file-action@v1.0.3
+#      with:
+#        file-url: "${{ matrix.config.assets_url }}"
+#        file-name: assets.7z
+#
+#    - name: Unpack assets
+#      shell: bash
+#      run: |
+#        mkdir -p assets
+#        cd assets
+#        7z x ../assets.7z
+#        cd ..
+#        rm assets.7z
+
+    - name: Configure
+      shell: bash
+      run: |
+        export PATH=$VITASDK/bin:$PATH
+        cmake -B build -G "${{ matrix.config.generator }}" -DCMAKE_BUILD_TYPE=${{ matrix.config.build_type }} ${{ matrix.config.extra_options }} .
+
+    - name: Build
+      shell: bash
+      run: |
+        export PATH=$VITASDK/bin:$PATH
+        cmake --build build --target all -j4
+
+    - name: Create Package
```

Comment:
There are few fixes need to apply to fix the pull-requests packaging. This problem was at all my scripts, but I fixed it. Need to backport that fix to here too. That's my fault I forgot that this script still have a packaging error at PRs.

Reply:
Done